Add Dashboard Access and Profiling Page (#2655)
pingcap/docs: dashboard/dashboard-access.md

---
title: Access TiDB Dashboard
summary: Learn how to access TiDB Dashboard.
category: how-to
---

# Access TiDB Dashboard

To access TiDB Dashboard, visit <http://127.0.0.1:2379/dashboard> via your browser. Replace `127.0.0.1:2379` with the actual PD instance address and port.

> **Note:**
>

> > If you have deployed multiple PD components in your cluster, you can use the address of only one PD component to access TiDB Dashboard, because the Dashboard operates on only one PD component. If you access via other PD components, your browser is redirected to that PD component. Therefore, if the firewall is not configured for this instance on which PD operates, you might not be able to access TiDB Dashboard.

Reviewer: Please check [the latest Chinese version](https://docs.pingcap.com/zh/tidb/dev/dashboard-access#%E5%A4%9A-pd-%E5%AE%9E%E4%BE%8B%E8%AE%BF%E9%97%AE) and update it.

Author: The updated part contains links of several documents added in https://github.com/pingcap/docs-cn/pull/3440. For convenience, I'll update the change made to `dashboard-access.md` in another PR together with those newly added documents.

Reviewer: Got it.

## Browser compatibility

> You can use TiDB Dashboard in the following desktop browsers which are common and updated timely:

Reviewer: ```suggestion
You can use TiDB Dashboard in the following common desktop browsers of a relatively newer version:
```

- Chrome >= 77
- Firefox >= 68
- Edge >= 17

> **Note:**
>

> > If you use the browsers above of earlier versions or other browsers to access TiDB Dashboard, some interfaces might not work properly.

Reviewer 2: ```suggestion
> If you use the browsers above of earlier versions or other browsers to access TiDB Dashboard, some functionalities might not work properly.
```

> ## Login

Reviewer 2: ```suggestion
## Sign in
```

Maybe better to align with the Sign In title in UI?

> For the first-time access, TiDB Dashboard displays the user login interface, as shown in the image below. You can log in using the TiDB `root` account.

Reviewer 2: ```suggestion
For the first-time access, TiDB Dashboard displays the user sign in interface, as shown in the image below. You can sign in using the TiDB `root` account.
```

![Login interface](/media/dashboard/dashboard-access-login.png)

> If the one of the following situations exists, the login might fail:

Reviewer: ```suggestion
If one of the following situations exists, the login might fail:
```

- TiDB `root` user does not exist.
- PD is not started or cannot be accessed.
- TiDB is not started or cannot be accessed.
- Wrong `root` password.

> Once you have logged in, the login status remains within the next 24 hours. To learn how to log out, refer to the [Log out](#log-out) section.

Reviewer 2: ```suggestion
Once you have signed in, the session remains valid within the next 24 hours. To learn how to sign out, refer to the [Logout](#logout) section.
```

## Switch language

The following languages are supported in TiDB Dashboard:

> - Chinese (simplified)
> - English

Reviewer: ```suggestion
- English
- Chinese (simplified)
```

> In the login interface, you can click the **Switch Language** drop-down box to switch the interface language.

Reviewer 2: ```suggestion
In the sign in page, you can click the **Switch Language** drop-down box to switch the interface language.
```

> In the **sign in** page, you can click the **Switch Language** drop-down box to switch the interface language.

Reviewer: ```suggestion
In the **SQL User Sign In** page, you can click the **Switch Language** drop-down list to switch the interface language.
```
